### PR TITLE
create AAQ resource

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

File: ocp_resources/aaq.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# API reference: https://github.com/kubevirt/application-aware-quota/blob/main/README.md
+# TODO: update API reference when OCP doc is available
+
+from ocp_resources.resource import Resource
+
+
+class AAQ(Resource):
```
Comment (myakove) [inline ocp_resources/aaq.py:7]: Is this resource that the user can create as well?

Comment (dbasunag) [reply ocp_resources/aaq.py:7]: I believe so - this work is going to evolve and provide more clarity in next few months. For our use case, it's deployment is currently controlled via HCO and this support was added just now.

Comment (myakove) [reply ocp_resources/aaq.py:7]: Please add TODO, to fill the missing code for create (to_dict())

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/aaq.py (hunk 1 above)
Comment (dbasunag) [reply ocp_resources/aaq.py:7]: Added one todo.

**Review by rnetser — APPROVED** [review]

**Review by myakove — APPROVED** [review]

**Pull request merged**